### ISSUE TITLE
These Op Not Support: Tensorflow::FusedBatchNorm

### ISSUE THREAD

**Issue by wangpupanjing** [alibaba/MNN#594]
ubuntu18.04

cd MNN/
./schema/generate.sh
mkdir build
cd build
cmake .. -DMNN_BUILD_CONVERTER=true && make -j4

Converte Tensorflow's Op tower_0/ShuffleNetV2/Stage2/unit_3/conv1x1_before/BatchNorm/cond/FusedBatchNorm , type = FusedBatchNorm, failed, may be some node is not const
Don't support not const scale node 
Converte Tensorflow's Op tower_0/ShuffleNetV2/Stage3/unit_1/conv1x1_before/BatchNorm/cond/FusedBatchNorm , type = FusedBatchNorm, failed, may be some node is not const
Don't support not const scale node 
Converte Tensorflow's Op tower_0/ShuffleNetV2/Stage2/unit_2/depthwise/BatchNorm/cond/FusedBatchNorm , type = FusedBatchNorm, failed, may be some node is not const
Don't support not const scale node 
Converte Tensorflow's Op tower_0/ShuffleNetV2/Stage2/unit_3/conv1x1_before/BatchNorm/cond/FusedBatchNorm_1 , type = FusedBatchNorm, failed, may be some node is not const
Don't support not const scale node 
Converte Tensorflow's Op tower_0/ShuffleNetV2/Stage3/unit_1/conv1x1_after/BatchNorm/cond/FusedBatchNorm_1 , type = FusedBatchNorm, failed, may be some node is not const
Don't support not const scale node 
Converte Tensorflow's Op tower_0/ShuffleNetV2/Stage2/unit_1/depthwise/BatchNorm/cond/FusedBatchNorm_1 , type = FusedBatchNorm, failed, may be some node is not const
[15:10:25] /home/wangpu/MNN/tools/converter/source/optimizer/PostConverter.cpp:20: Inputs: training_flag
[15:10:25] /home/wangpu/MNN/tools/converter/source/optimizer/PostConverter.cpp:20: Inputs: tower_0/images
[15:10:25] /home/wangpu/MNN/tools/converter/source/optimizer/PostConverter.cpp:37: Outputs: tower_0/SpatialSqueeze, Type = Squeeze
The Model Has Control / Extra Op, Please Compile the Code of model.cpp
terminate called after throwing an instance of 'Error'
  what():  [15:10:25] /home/wangpu/MNN/tools/converter/source/common/writeFb.cpp:107: Check failed: (notSupportOps.size()) == (0) ==> These Op Not Support: Tensorflow::FusedBatchNorm | 
已放弃 (核心已转储)

**Comment by jxt1234** [comment]
请在导出 pb 时把 is_training 设成 false

**Comment by liuxx2016** [comment]
@jxt1234大佬您好， 如何在导出pb时将is_training设为false？在哪设置？
